src/fsharp/ConstraintSolver.fsi

```diff
@@ -90,49 +90,94 @@ type ContextInfo =
     | SequenceExpression of TType
 
 /// Captures relevant information for a particular failed overload resolution.
-type OverloadInformation = 
-    {
-        methodSlot: CalledMeth<Expr>
-        infoReader: InfoReader
-        error: exn
-    }
+type OverloadInformation =
+    { methodSlot: CalledMeth<Expr>
+      infoReader: InfoReader
+      error: exn }
 
 /// Cases for overload resolution failure that exists in the implementation of the compiler.
 type OverloadResolutionFailure =
-  | NoOverloadsFound   of methodName: string
-                        * candidates: OverloadInformation list 
-                        * cx: TraitConstraintInfo option
-  | PossibleCandidates of methodName: string 
-                        * candidates: OverloadInformation list // methodNames may be different (with operators?), this is refactored from original logic to assemble overload failure message
-                        * cx: TraitConstraintInfo option
+    | NoOverloadsFound of methodName: string * candidates: OverloadInformation list * cx: TraitConstraintInfo option
+    | PossibleCandidates of
```

Comment:
Another place where an empty line between multiline declarations would look great (as discussed in https://github.com/fsprojects/fantomas/issues/2198).

Reply:
Could be pitched in the style guide.